app/views/layouts/application.html.haml

```diff
@@ -22,6 +22,7 @@
     - unless Rails.env.test?
       - Webpacker::Manifest.instance.data.keys.each do |pack|
         = javascript_pack_tag pack if pack.ends_with? '-common.js'
+    = javascript_pack_tag "manageiq-graphql/graphql-explorer"
```

Comment:
Looks great @imtayadeway !

The only question I have is - should `javascript_pack_tag "manageiq-graphql/graphql-explorer"` be included under `Rails.env.development?` conditional?

Reply:
Ooh that's a great question and I think you are correct. Will update.

I had a wider question as I was writing this, wondering if this was the correct place to include this tag?